src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -0,0 +1,76 @@
+package frc.robot.subsystems;
```

Comment:
comment in general, not for this specific line:
you dont seem to use the hall effects at all. not in here, and not in the command